epersons.md

```diff
@@ -55,4 +56,56 @@ the replace operation `[{ "op": "replace", "path": "/password", "value": "newpas
 ```json
   "password": "newpassword",
 ```
-Note: The new password is currently returned after an update but this could be revisited later, see [#30]((https://github.com/DSpace/Rest7Contract/issues/30))
\ No newline at end of file
+Note: The new password is currently returned after an update but this could be revisited later, see [#30]((https://github.com/DSpace/Rest7Contract/issues/30))
+
+To replace the firstname and lastname metadata fields, `curl -X PATCH http://${dspace.url}/api/epersons/eperson/<:id-eperson> -H Content-Type: application/json" -d '[{ "op": "replace", "path": "metadata/eperson.firstname", "value": "newName"]`. The operation also requires an Authorization header.
```

Comment:
The #46 is the right direction here, so we will need only to reference the general metadata patch documentation here